drv/stm32h7-eth/src/lib.rs

```diff
@@ -387,6 +391,7 @@ pub enum SmiClause22Register {
     MmdAccessControl = 13,
     MmdAccessAddressData = 14,
     ExtendedStatus = 15,
+    TxFxExtendedStatus = 16,
```

Comment:
This register does not appear in 802.3 clause 22, at least in the version I've got. Is this a vendor-specific extension?

Reply:
Yes, this is a vendor-specific extension.  I've removed in favor of using registers from the VSC7448 PAC, which is already used in the BSP that was using `TxFxExtendedStatus`.

I'm not totally happy having the definitions in two places, though.

[Here's the main definitions in the PAC](https://github.com/oxidecomputer/vsc7448/blob/master/vsc7448-pac/src/phy.rs#L44), and [here's an example of register structs](https://github.com/oxidecomputer/vsc7448/blob/master/vsc7448-pac/src/phy/standard.rs#L32).

We should chat about this when we discuss live.  We could remove `eth::SmiClause22Register` entirely; the downside would be compiling an extra relatively heavy crate (once).

Reply:
When talked live, the consensus was to preserve `eth::SmiClause22Register` as a lightweight way to talk to registers.  I added a detailed comment suggesting using the PAC register addresses if that crate is pulled in.